lightly_studio_view/src/lib/components/TagCreateDialog/TagCreateDialog.svelte

```diff
@@ -27,28 +27,34 @@
     };
 
     let { datasetId, gridType }: UseTagsCreateDialog = $props();
-    let tagKind: TagKind = $derived(gridType === 'samples' ? 'sample' : 'annotation');
-
+    let tagKind: TagKind = $derived(
+        ['samples', 'videos', 'video_frames'].includes(gridType) ? 'sample' : 'annotation'
+    );
     const { tags, loadTags } = $derived(useTags({ dataset_id: datasetId, kind: [tagKind] }));
 
     // setup global selection state
     const {
-        selectedSampleIds,
+        getSelectedSampleIds,
         selectedSampleAnnotationCropIds,
         clearSelectedSampleAnnotationCrops,
         clearSelectedSamples
     } = useGlobalStorage();
+    const selectedSampleIds = getSelectedSampleIds(datasetId);
```

Comment:
```suggestion
    const selectedSampleIds = $derived(getSelectedSampleIds(datasetId));
```

otherwise this is only called once when the component initializes, and the video dataset id would always be used (for example, the create tag button would be disabled for frames until you also select some videos in the video grid)

Reply:
done